app/models/application_pet.rb

```diff
@@ -1,4 +1,8 @@
 class ApplicationPet < ApplicationRecord
   belongs_to :application
   belongs_to :pet
+
+  def approve!
```

Comment:
so should have just added a method to applicationpet model and updated our db:columns to add status for applicationspets table